website_logo/views/website_templates.xml

```diff
@@ -0,0 +1,13 @@
+<odoo>
+    <template id="add_logo_to_website" inherit_id="website.layout_logo_show" customize_show="False" name="Show Logo">
+        <xpath expr="//header//a[hasclass('navbar-brand')]" position="after">
+            <a href="/" class="navbar-brand logo">
+                <span t-field="website.logo" t-options="{'widget': 'image'}" role="img" t-att-aria-label="'Logo of %s' % website.name" t-att-title="website.name"/>
+            </a>
+        </xpath>
+        <xpath expr="//header//a[hasclass('navbar-brand')]" position="attributes">
+            <attribute name="class" add="invisible" separator=" "/>
```

Comment:
Here can use 'd-none' (bootstrap4) instead of 'invisible'... this hidden the element in the layout.